app/common.py

```diff
@@ -7,6 +7,7 @@
 from .user_api.errors import UserAuthenticationError
 from werkzeug.exceptions import Unauthorized
 import json
+from typing import Dict, Any
 
 
 class FirebaseDataEntity(ABC):
```

Comment:
I'm cool with ditching this class, but the to_json function needs to be added back to whatever data classes were inheriting from it before since this was being used.

Reply:
On second thought, this is basically only used in the test so it's not necessarily needed

Reply:
If it's okay I'll leave the removal to you so you can de-couple your stuff and I don't want to add more to the scope of this PR.